src/Cards.cpp

```diff
@@ -0,0 +1,18 @@
+#include "Cards.hpp"
+
+Cards::Cards(const std::string & color, json dataPacked) :
```

Comment:
Why not an enum? Color can have only 2 values.
Good idea with taking json as a constructor parameter :)